### PR TITLE
Grammar+spelling asn_1990s_spectator_of_democracy

### PR DESCRIPTION
**Pull request by Devices**
I chose a pull request, because I am not a native speaker and some of
the changes might be wrong and/or controversial. Let me know what
you think.

flip-side > flip side. Flip-side does not appear to exist in any dictionary.
(line 63-64)

market place -> marketplace. This is always one word as far as I know.
And they used the correct form twice elsewhere in the text. (line 71)

cat-calls -> catcalls. The hyphenated version does not appear to exist.
(line 75)

"Of course presidential elections are only". Added a comma after the
introductory word 'of course'. (line 89 and 98)

counter-part -> counterpart. Another non-existing hyphenated version. (line 118)

"free-reign" -> free rein. Non-existing hyphenated form. Controversial change:
reign to rein. In this context they mean allowed freedom and not to rule over.
References: https://www.merriam-webster.com/words-at-play/usage-free-rein-vs-free-reign
and https://grammarist.com/spelling/free-rein-free-reign/. (line 148)

"So democracy is an ideal strategy." Added comma after introductory word
'so'. (line 158)

"Thus the ruling party switched from the party of corruption". Comma after
thus. (line 162)

"But naturally democracy implies many more switches after this." I think a
comma is needed after 'But naturally' and have thus placed it. (line 164)

"(There was some cheating but we already know cheating is part of any game.)"
Needs a comma before 'but' (line 178)

"Thus the final end of the Soviet Union". Comma again. (line 193)

a profound disagreements -> a profound disagreement. Plural does not make
grammatical sense here. (line 255)

blue-print -> blueprint. Blue-print does not appear in any dictionary.
(line 264)

"Thus democratic ideology creates..." -> introduce comma after thus, an
introductory word. (line 270)

"The formal decision making process will depend" decision making ->
decision-making. This seems to be correct for both english and american-
english. (line 280)

"(Again there was certainly a lot of cheating in the French CGT's". Added
a comma after the introductory word 'again'. (line 310)

"At the point when revolutionaries realize that they have nothing to lose
from the destruction of this society, they may realize the mirage of
it's democracy." it's -> its. This appears to be the possessive form and
not either it is or it has. (line 344)

### CODE REVIEW

**Comment by redschalken** [comment]
tl;dr: I'll defer to you and xat on this, but I am against adding the commas. It is too much a matter of personal preference; clearly the original author didn't think they were necessary (and shouldn't we respect their wishes?). What's more, it's a lot of work to go back and add or remove commas, and we could even get to a situation where some contributors add commas, while other contributors remove them, and then another set of contributors add them back.

That said, I *am* in favor of the hyphenation fixes and the grammatical fix for line 255.

I'm okay with changing 'free-reign' to 'free rein' as well. This change looks big but it's basically just another spelling fix.

Full reply:

Thanks for finding these issues, carefully explaining your reasoning, and for continuing to contribute!

Your pull request raises the question: in what ways do we want to change a text?

I have avoided modifying punctuation except when it's an obvious error. If you search the commit history for mentions of commas, you'll see that almost every change that mentions commas is along these lines:

* "Remove spaces in front of two commas."

* "... replace a comma at the end of a sentence with a period"

* "Change a semi-colon to a comma and fix some of the bibliography. Working from a a PDF scan of the original."

* "Add comma, as confirmed by the German and Norwegian versions."

So far, I have mostly limited myself to minor changes like these:

* Converting the source (print or HTML) to markdown (examples: dropping images, changing underlining to bold, simplifying layouts, removing tabs or excessive whitespace).

* Correcting obvious errors caused by OCR or bad transcriptions

* Changing some non-ascii characters to ascii substitutes (ex: 'ö' to 'oe'.  I stopped doing this early on, and may have went back and put back in the non-ascii characters in some files)

* Used uniform spellings for some names (e.g., changed all variant spellings of Khrushchev, changed 'Roumania' or 'Rumania' to 'Romania').

* In Nicolaievsky's Karl Marx: Man and Fighter, the original text had French quotes in the body of the text, and English translations as endnotes. In our redtexts version I swapped the English into the body and put the French quotes in the endnotes.

* There might be cases where I fixed a misspelling, but at least in some texts -- particularly those that I scanned and for which there was never a previous online version -- I left in the errors and indicated them with [sic].

In any case, we might think about how much we want to change in a text, and then document our decision in the readme.

**Comment by Devices** [comment]
>tl;dr: I'll defer to you and xat on this, but I am against adding the commas. It is too much a matter of personal preference; clearly the original author didn't think they were necessary (and shouldn't we respect their wishes?). What's more, it's a lot of work to go back and add or remove commas, and we could even get to a situation where some contributors add commas, while other contributors remove them, and then another set of contributors add them back.
I think it is not necessarily the author's wishes. It looks like they either do not master the language well or they had no editor. To me this is obvious from the use of 'market place' one, and 'marketplace'  twice in the text and the common mistake of confusing the possessive 'its' and subject+verb 'it's'. It does indeed raise the question if we should be the editor instead, because I certainly am not qualified to. Let alone the load on contributors to edit all the texts and, as you rightly state, the disagreements about certain grammar points or comma placement.

That brings us to the next issue:
>Your pull request raises the question: in what ways do we want to change a text?

I think fixing some small spelling and grammar errors would improve the readability of a text, such as missing double letters (could be an OCR error too) or "its" <-> "it's" errors. I think adding [sic] to a text is a more significant change to the original than just fixing the error. I understand your argument that changing a text from your original scan might not be ideal, because there is no original (online) version to obtain. There is also no guarantee that versions with an original (online) version will stay online, so the redtext version might end up becoming the only version for some texts at some point in time. For this reason I think we should have a uniform guideline for all texts.

For the record, I could easily live without the commas. Changes I think add to the readability to the text are: the hyphens, line 71 (taking the text's English version in mind), line 255 and line 344.

This project does seem to be mainly focused on archiving text in a certain format, so editing is, at least historically, not a big part of the activities.

For reference some historical editing, I didn't want to look through them all. This is perhaps a good starting point for writing guidelines, although it is mostly covered by @redschalken's post already.
millenium -> millennium https://github.com/redtexts/texts/commit/dbb1f5d6d5fa508956e5602558637f04caa22b65
technic -> technique, posseses [sic] -> possesses, Wester [sic] -> Western, and some more https://github.com/redtexts/texts/commit/0b39a9d3e13bef0e4558e8dbbf945249949e3bab
councillists -> councilists, jewelery -> jewelry https://github.com/redtexts/texts/commit/2af89c631af19a61579736c434c8c872a82948b4
their -> there https://github.com/redtexts/texts/commit/0cd8f3e0987f65bdeb95fdc885f0a59c210f4a2a

**Comment by xatasan** [comment]
sorry for the late response, didn't get to check my mail. don't have
much to add, but a few comments nevertheless.

Devices <notifications@github.com> writes:

> flip-side > flip side. Flip-side does not appear to exist in any dictionary.
> (line 63-64)

I've seen both, but I perfer "flip-side" or even "flipside" over two
seperate words.

> market place -> marketplace. This is always one word as far as I know.
> And they used the correct form twice elsewhere in the text. (line 71)

agree.

> cat-calls -> catcalls. The hyphenated version does not appear to exist.
> (line 75)

agree.

> "Of course presidential elections are only". Added a comma after the
> introductory word 'of course'. (line 89 and 98)

undecided, but would also rather not add it.

> counter-part -> counterpart. Another non-existing hyphenated version. (line 118)

agree.

> "free-reign" -> free rein. Non-existing hyphenated form. Controversial change:
> reign to rein. In this context they mean allowed freedom and not to rule over.
> References: https://www.merriam-webster.com/words-at-play/usage-free-rein-vs-free-reign
> and https://grammarist.com/spelling/free-rein-free-reign/. (line 148)

seems like a typo, so ok.

> "So democracy is an ideal strategy." Added comma after introductory word
> 'so'. (line 158)

> "Thus the ruling party switched from the party of corruption". Comma after
> thus. (line 162)

> "But naturally democracy implies many more switches after this." I think a
> comma is needed after 'But naturally' and have thus placed it. (line 164)

> "(There was some cheating but we already know cheating is part of any game.)"
> Needs a comma before 'but' (line 178)

> "Thus the final end of the Soviet Union". Comma again. (line 193)

I think I agree with schalken, that these changes aren't necessary. But
if we could find the original text or scan, it might be worth checking
if mistakes were made during digitization.

> a profound disagreements -> a profound disagreement. Plural does not make
> grammatical sense here. (line 255)

agree.

> blue-print -> blueprint. Blue-print does not appear in any dictionary.
> (line 264)

ok.

> "Thus democratic ideology creates..." -> introduce comma after thus, an
> introductory word. (line 270)

ut supra

> "The formal decision making process will depend" decision making ->
> decision-making. This seems to be correct for both english and american-
> english. (line 280)

I don't think this is necessary, I've seen both.

> "(Again there was certainly a lot of cheating in the French CGT's". Added
> a comma after the introductory word 'again'. (line 310)

u.s.

> "At the point when revolutionaries realize that they have nothing to lose
> from the destruction of this society, they may realize the mirage of
> it's democracy." it's -> its. This appears to be the possessive form and
> not either it is or it has. (line 344)

agree.

-- 
		~xat
		https://tilde.town/~xat/

**Comment by Devices** [comment]
> I've seen both, but I perfer "flip-side" or even "flipside" over two seperate words.

I have a lot of trouble finding any instances of "flip-side" online. Flipside seems to occur occasionally, but flip side seems to be most used by a large margin. It is two separate words just like "opposite side" (synonym) is not "oppositeside". That is my take on it at least and dictionaries seem to agree so far. I do not think it has gone through a similar evolution as, for example, market place becoming marketplace. I think it should be either flipside or flip side, with my preference going to the latter. I will stay with my original choice for now, but please do discuss or edit.

>[...] commas [...]

Seeing your and Schalken's comments I have changed my pull request to change all commas. I think they are indeed too contentious to have as a rule for the project and redtexts should stay close to the source in this case.

>> "The formal decision making process will depend" decision making -> decision-making. This seems to be correct for both english and american- english. (line 280)
> I don't think this is necessary, I've seen both.

This, indeed, appears to be the case. I will edit the pull request as such.
